css/style.css

```diff
@@ -0,0 +1,156 @@
+*{
+  padding: 0;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+}
+
+nav{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 5px 0 30px;
+  height: 55px;
+}
+.yt-logo{
+  width: 120px;
+}
+
+.yt-search{
+  position: relative;
+  align-items: center;
+  border-radius: 2px 0 0 2px;
+  padding: 2px 6px;
+  margin-left: 34px;
+  flex: 1;
+  flex-basis: 1e-9px;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+}
+
+.ico{
+  width: 40px;
+}
+
+.yt-search input{
+  width: 100%;
+  padding: 10px;
+  margin: 0;
+  border: 1px solid #d3d3d3;
+  border-radius: 1px;
+  box-shadow: inset 0 1px 2px #f8f8f8;
+  color: #f8f8f8;
```

Comment:
Change color to a more visible one.

Reply:
- [OPTIONAL] Nice one @SekoViper, but try as much as you can to include `WHY` the student is asked to make use of a suggestion as this would make it clearer for the student to know why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where more information could be got for  more clarification.